NO-JIRA support using gateway in different namespace in HTTPRoute

alyson-ferreira-trimble opened the pull request:
Improvement: Support using a Gateway in a different namespace

- **feat(httproute): support gateway in different namespace**
- **chore(httproute): update tests**

Please ensure your pull request adheres to the following guidelines:
- [x] explain your motives to contribute this change: what problem you are trying to fix, what improvement you are trying to make
- [x] Document your Changes in the `CHANGELOG.md` file of the respected chart as well as the `Chart.yaml`

jCOTINEAU commented:
Hello @alyson-ferreira-trimble thanks a lot for the contribution in the community, you can rebase from master as the 10.7 released happened, you can squeeze all your changes directly in the 10.8.0 version, as we ship one chart version per SonarQube version hence, all changes are bundled into the same release.

in the meantime I will test locally

alyson-ferreira-trimble commented:
> Hello @alyson-ferreira-trimble thanks a lot for the contribution in the community, you can rebase from master as the 10.7 released happened, you can squeeze all your changes directly in the 10.8.0 version, as we ship one chart version per SonarQube version hence, all changes are bundled into the same release.
> 
> in the meantime I will test locally, but you can also change the files in `tests/unit-compatibility-test/sonarqube(-dce)/http-routes(-default)-values.yaml` and add the namespace parameter

@jCOTINEAU I've rebased and added the tests. Thanks!

jCOTINEAU commented:
Hello @alyson-ferreira-trimble Huge thanks for your contribution, i will close your PR in favor of this [one](https://github.com/SonarSource/helm-chart-sonarqube/pull/579/files)

Some of our CI tests work only when an employee creates the PR. Nonetheless, you'll still be marked as the author and contributor of your commits and we really want to say thank you for the contributions.
